test/plots/rectanglePlotTests.ts

```diff
@@ -2,511 +2,486 @@
-      assert.strictEqual(cells.length, data.length);
-      for (let i = 0; i < cells.length; i++) {
-        let cell = d3.select(cells[i]);
-        assert.strictEqual(cell.attr("x"), String(i * CELL_WIDTH), "Cell x coord is correct");
-        assert.strictEqual(cell.attr("y"), String(i * CELL_HEIGHT), "Cell y coord is correct");
-        assert.strictEqual(cell.attr("width"), String(CELL_WIDTH), "Cell width is correct");
-        assert.strictEqual(cell.attr("height"), String(CELL_HEIGHT), "Cell height is correct");
-      }
-      svg.remove();
-    });
 
-    it("can invert y axis correctly", () => {
-      let xScale = new Plottable.Scales.Category();
-      let yScale = new Plottable.Scales.Category();
-      let colorScale = new Plottable.Scales.InterpolatedColor();
-      colorScale.range(["black", "white"]);
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let gridPlot = new Plottable.Plots.Rectangle();
-      gridPlot.addDataset(new Plottable.Dataset(DATA))
-              .attr("fill", (d) => d.magnitude, colorScale);
-      gridPlot.x((d: any) => d.x, xScale)
-              .y((d: any) => d.y, yScale)
-              .renderTo(svg);
-
-      yScale.domain(["U", "V"]);
-
-      let cells = (<any> gridPlot)._renderArea.selectAll("rect")[0];
-      let cellAU = d3.select(cells[0]);
-      let cellAV = d3.select(cells[2]);
-      cellAU.attr("fill", "#000000");
-      cellAU.attr("x", "0");
-      cellAU.attr("y", "100");
-
-      cellAV.attr("fill", "#ffffff");
-      cellAV.attr("x", "0");
-      cellAV.attr("y", "0");
-
-      yScale.domain(["V", "U"]);
-      cells = (<any> gridPlot)._renderArea.selectAll("rect")[0];
-      cellAU = d3.select(cells[0]);
-      cellAV = d3.select(cells[2]);
-      cellAU.attr("fill", "#000000");
-      cellAU.attr("x", "0");
-      cellAU.attr("y", "0");
-
-      cellAV.attr("fill", "#ffffff");
-      cellAV.attr("x", "0");
-      cellAV.attr("y", "100");
-
-      svg.remove();
-    });
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Category;
+      let yScale: Plottable.Scales.Category;
+      let colorScale: Plottable.Scales.InterpolatedColor;
+      let plot: Plottable.Plots.Rectangle<string, string>;
+      let dataset: Plottable.Dataset;
 
-    describe("selections()", () => {
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        xScale = new Plottable.Scales.Category();
+        yScale = new Plottable.Scales.Category();
+        colorScale = new Plottable.Scales.InterpolatedColor();
+        colorScale.range(["black", "white"]);
+
+        plot = new Plottable.Plots.Rectangle<string, string>();
+        plot.attr("fill", (d) => d.magnitude, colorScale);
+        plot.x((d: any) => d.x, xScale);
+        plot.y((d: any) => d.y, yScale);
+
+        dataset = new Plottable.Dataset(data);
+        plot.addDataset(dataset);
+        plot.renderTo(svg);
```

Comment:
Because all this behavior is tested already above. Do you think it is too specific?